src/js/ripple/log.js

```diff
@@ -149,7 +149,11 @@ var NullLogEngine = {
 if (typeof window !== 'undefined' && typeof console !== 'undefined') {
   Log.engine = InteractiveLogEngine;
 } else if (typeof console !== 'undefined' && console.log) {
-  Log.engine = BasicLogEngine;
+  if (process.env.NODE_ENV === 'test') {
+    Log.engine = NullLogEngine;
```

Comment:
This is going to suppress even error messages. If they are intentional errors in the tests, then maybe it would be safer to switch to the NullLogEngine only for the duration of those specific tests and have the BasicLogEngine reinstalled before each test case. Otherwise it could hide error messages that we weren't expecting.